pkgs/development/libraries/gstreamer/bad/default.nix

```diff
@@ -11,6 +11,7 @@
 , orc
 , gstreamer
 , gobject-introspection
+, enableOpencv ? false
```

Comment:
This might be a bit too nitpicky, but could you rename this to `opencvSupport` and move it down around where `microdnsSupport` is?

Most of the plugin-based switches use `<plugin>Support` rather than `enable<Plugin>`

Otherwise this looks good for me, thank you!